cryptol-saw-core/src/Verifier/SAW/Cryptol.hs

```diff
@@ -332,6 +332,45 @@ isErasedProp prop =
     C.TCon (C.PC C.PLiteralLessThan) _ -> False
     _ -> True
 
+-- | Translate an erased 'Prop' to a 'Term'
```

Comment:
How about a little more specificity, by saying something like: Translate an erased `Prop` to Boolean `Term` that tests if the `Prop` holds; this will `panic` for `Prop`s that cannot be effectively decided, such as (and then give an example)

Reply:
Also, the fact that the `Prop` is erased seems to be a red herring. Instead, maybe call this `importPropToBool` or something like that? This is in fact meant as a question, and I'm open to you saying "no", so please say if you think the erased-ness is important.

Reply:
I agree, the erased-ness isn't the important part here. I took this even further and renamed it to `importNumericConstraintAsBool` and added some more detailed documentation about what it can and can't take as inputs.